### ISSUE TITLE
[9.1.0.288 for 1.10.2] CRITICAL game crash crafting a solar generator in Tinkers crafting table.

### ISSUE THREAD

**Issue by ecksfiftyone** [aidancbrady/Mekanism#3578]
[crash-2016-07-31_16.51.17-client.txt](https://github.com/aidancbrady/Mekanism/files/392640/crash-2016-07-31_16.51.17-client.txt)

Tried to craft a solar generator in a Tinkers crafting table. The game crashed and now the components are still in the crafting table. Any time anyone goes within 20 - 30 blocks of the crafting table their game crashes and they are unable to rejoin the server since they spawn in the someplace they crashed. I have tried to recover. I will try disabling all mods I can, but at this point it seems that I will need to scrap my server and start fresh. I'd call this a serious bug. 

Details:
	Minecraft Version: 1.10.2
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_25, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 913634224 bytes (871 MB) / 2567307264 bytes (2448 MB) up to 4241293312 bytes (4044 MB)
	JVM Flags: 6 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx4G -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -Xmn512M
	IntCache: cache: 0, tcache: 0, allocated: 13, tallocated: 95
	FML: Minecraft 1.10.2 MCP 9.32 Powered by Forge 12.18.0.2006 Optifine OptiFine_1.10.2_HD_U_C2 32 mods loaded, 32 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
	UCHIJA	mcp{9.19} [Minecraft Coder Pack] (minecraft.jar) 
	UCHIJA	FML{8.0.99.99} [Forge Mod Loader] (forge-1.10.2-12.18.0.2006-1.10.0.jar) 
	UCHIJA	Forge{12.18.0.2006} [Minecraft Forge] (forge-1.10.2-12.18.0.2006-1.10.0.jar) 
	UCHIJA	armorplus{1.9.4-4.0.1.0} [ArmorPlus] (ArmorPlus1.9.4-4.0.1.0.jar) 
	UCHIJA	Chameleon{1.9.4-2.1.5} [Chameleon] (Chameleon-1.9.4-2.1.5.jar) 
	UCHIJA	cxlibrary{1.2.1} [CXLibrary] (cxlibrary-1.10-1.2.1.jar) 
	UCHIJA	FastLeaveDecay{1.1} [Fast Leave Decay] (FastLeaveDecay-MC1.9.4-1.2.1.jar) 
	UCHIJA	gravestone{1.5.1} [Gravestone] (GraveStone Mod 1.5.1.jar) 
	UCHIJA	hopperducts{1.4.7} [Hopper Ducts] (hopperducts-mc1.9.4-1.4.7.jar) 
	UCHIJA	InventoryPets{1.4.8} [Inventory Pets] (inventorypets-1.10-1.4.8.jar) 
	UCHIJA	inventorytweaks{1.61-58-a1fd884} [Inventory Tweaks] (InventoryTweaks-1.61-58.jar) 
	UCHIJA	IronChest{7.0.3.791} [Iron Chest] (ironchest-1.10-7.0.3.791.jar) 
	UCHIJA	jacb{1.0.6} [JACB] (JACB-1.10.2-1.0.6.jar) 
	UCHIJA	JEI{3.7.5.227} [Just Enough Items] (jei_1.10.2-3.7.5.227.jar) 
	UCHIJA	fyberbackpack{1.0.2} [Just Backpacks] (justbackpacks-mc1.9.4-1.0.2.jar) 
	UCHIJA	mantle{1.10-0.10.3.jenkins156} [Mantle] (Mantle-1.10-0.10.3.jar) 
	UCHIJA	mcmultipart{1.2.0_72} [MCMultiPart] (MCMultiPart-experimental-1.2.0_72-universal.jar) 
	UCHIJA	Mekanism{9.1.0} [Mekanism] (Mekanism-1.10.2-9.1.0.288.jar) 
	UCHIJA	MekanismGenerators{9.1.0} [MekanismGenerators] (MekanismGenerators-1.10.2-9.1.0.288.jar) 
	UCHIJA	MekanismTools{9.1.0} [MekanismTools] (MekanismTools-1.10.2-9.1.0.288.jar) 
	UCHIJA	MoreFurnaces{1.8.1} [More Furnaces] (MoreFurnaces-1.10-1.8.1.jar) 
	UCHIJA	progressiveautomation{1.6.48} [Progressive Automation] (ProgressiveAutomation-1.10.2-1.6.48.jar) 
	UCHIJA	reborncore{2.5.4.47} [RebornCore] (RebornCore-1.9.4-2.5.4.47-universal.jar) 
	UCHIJA	reborncore-mcmultipart{2.5.4.47} [reborncore-MCMultiPart] (RebornCore-1.9.4-2.5.4.47-universal.jar) 
	UCHIJA	StorageDrawers{1.9.4-3.1.10} [Storage Drawers] (StorageDrawers-1.9.4-3.1.10.jar) 
	UCHIJA	refinedstorage{0.8.2} [Refined Storage] (refinedstorage-0.8.2.jar) 
	UCHIJA	tconstruct{1.10-2.3.3a.jenkins271} [Tinkers' Construct] (TConstruct-1.10-2.3.3a.jar) 
	UCHIJA	techreborn{1.3.2.120} [TechReborn] (TechReborn-1.9.4-1.3.2.120-universal.jar) 
	UCHIJA	TinkersAddons{1.0.3} [Tinkers' Addons] (Tinkers' Addons-1.10.x-1.0.3.jar) 
	UCHIJA	Waila{1.7.0} [Waila] (Waila-1.7.0-B3_1.9.4.jar) 
	UCHIJA	WailaHarvestability{1.1.7} [Waila Harvestability] (WailaHarvestability-mc1.10-1.1.7.jar) 
	UCHIJA	weaponsplus{1.9.4-3.0.0.0} [WeaponsPlus] (WeaponsPlus1.9.4-3.0.0.0.jar) 
	Loaded coremods (and transformers): 
LoadingHook (Mekanism-1.10.2-9.1.0.288.jar)

**Comment by MrPatience** [comment]
Same thing with a mekanism tank , it's about rendering (btw your crash report is somewhat uncomplete)
Until fix, you should avoid using crafting station when crafting mekanism stuff. Also , to fix it on your world , rollback your world or : Get an Op far away from crash area , give him crafting station coordinates (they are on the crash report) and /setblock

**Comment by ecksfiftyone** [comment]
You sir, are a genius. You saved my server from a reset, I forgot all about
/setblock. I issued this from console and all is well.  Oddly enough, I
expected this to be the crafting table, but it ended up being Tinker's
Smeltery controller that was replaced?.. either way, it's all good now.

I donated a few bucks, and will do so from time time time. Mechanism is my
favorite mod hands down.

I assume you are aware also that breaking a cable or pressurized tube with
a tinkers or vanilla minecraft tool crashes the game as well. I got around
this by using a Paxel to break these.

Thanks again!
